File: ios/Classes/Models/MPredictions.swift
//
//  MPredictions.swift
//  places_autocomplete
//
//  Created by Diwakar Singh on 27/09/22.
//

import Foundation

struct MPredictions: Encodable {
    let autocompletePredictions: [MPrediction]?
}

struct MPrediction: Encodable {
    let placeId: String?
    let distanceMeters: Int?
    let description: String?
    let primaryText: String?
    let secondaryText: String?
    let types: [String]?
}

File: android/src/main/kotlin/co/codezens/places_autocomplete/models/MPredictions.kt
package co.codezens.places_autocomplete.models

data class MPredictions(
    val autocompletePredictions: List<MPrediction>?
)

data class MPrediction(
    val placeId: String?,
    val distanceMeters: Int?,
    val description: String?,
    val primaryText: String?,
    val secondaryText: String?,
    val types: List<String>?
)